zio-http/src/main/scala/zhttp/http/Cookie.scala

```diff
@@ -233,7 +238,7 @@ object Cookie {
           domain = headerValue.substring(curr + 7, next)
         } else if (headerValue.regionMatches(true, curr, fieldPath, 0, fieldPath.length)) {
           val v = headerValue.substring(curr + 5, next)
-          if (!v.isEmpty) {
+          if (v.nonEmpty) {
```

Comment:
Changed these since I was already working in the file, and v.nonEmpty is more readable than !v.isEmpty